tools/parse-release-notes

```diff
@@ -38,7 +38,7 @@ const formatNotes = ({ notes, language, production, platformName }) => {
     })
     .join('\n')
 
-  const notesWithDefault = `${defaultNote}\n${formattedNotes}`
+  const notesWithDefault = `${defaultNote}${formattedNotes}`
```

Comment:
Are you sure you do not need the newline here? join only inserts the newline in between the elements, dosn't it?